esp_delta_ota/examples/https_delta_ota/tools/esp_delta_ota_patch_gen.py

```diff
@@ -19,40 +23,115 @@
 # Magic Byte is created using command: echo -n "esp_delta_ota" | sha256sum
 esp_delta_ota_magic = 0xfccdde10
 
-MAGIC_SIZE = 4
-DIGEST_SIZE = 32
-RESERVED_HEADER = 64 - (MAGIC_SIZE + DIGEST_SIZE)
+MAGIC_SIZE = 4 # This is size of magic byte
+DIGEST_SIZE = 32 # This is SHA256 of the base binary    
+CRC_SIZE = 4 # This is CRC32 of patch file which shall be applied on the base binary
+RESERVED_HEADER = 64 - (MAGIC_SIZE + DIGEST_SIZE + CRC_SIZE) # This is reserved header size
```

Comment:
I think we can also add a HEADER_VERSION field here,
This field shall be the version of the patch header.
In the esp_delta_ota component, we shall check this version as follows

if (version > 0) {
   verify crc32 integrity of the patch
}

Reply:
Can we define a new field called `HEADER_SIZE = 64` and use it instead of the value `64`

Reply:
> I think we can also add a HEADER_VERSION field here, This field shall be the version of the patch header. In the esp_delta_ota component, we shall check this version as follows
> 
> if (version > 0) { verify crc32 integrity of the patch }

Yep, we can add but where this field should be added, at the beginning of the header or at last. Because I don't think is there any need of specific locations. What's your opinion?